packages/functionals/botpress-qna/README.md

```diff
@@ -77,6 +77,16 @@ const questionsFlatExported = await bp.qna.export({ flat: true })
 // ]
 ```
 
+## Controlling whether Q&A should intercept
+
+It may appear that it's not usefule for Q&A to just intercept all the users' messages and try to match them against Q&A's intents. This can be customized by providing a hook to Q&A module that will prevent interception when returning `false`.
```

Comment:
typo: `useful`

Reply:
Returning `false` _or a promise resolving to `false`_

Reply:
@emirotin , fixed that.